src/layout/MVP/LoginLayout/LoginLayout.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { GovBanner } from '@trussworks/react-uswds'
+import classnames from 'classnames'
+
+import styles from './LoginLayout.module.scss'
+
+const LoginLayout = ({ children }: { children: React.ReactNode }) => {
+  return (
+    <>
+      <a className="usa-skipnav" href="#main-content">
+        Skip to main content
+      </a>
+      <div className={styles.siteContainer}>
+        <div className={classnames(styles.mainContainer, styles.layoutLogin)}>
```

Comment:
I don't think this is a big deal because visually they look the same, but the login layout in MVP looks like it's just the skipnav link, banner & `<main>` tag, without siteContainer & mainContainer divs